src/components/Form.js

```diff
@@ -25,7 +25,7 @@ export default function Form() {
 
   return (
     <div className="add">
-      <h2>ADD A NEW BOOK</h2>
+      <h2>ADD NEW BOOK</h2>
       <form onSubmit={handleSubmit}>
```

Comment:
- [ ] In zeplin design, there is also a dropdown named 'category'. I am requesting you to add that as I am supposed to review according to the given design. If your prior milestone didn't require you to add this input field, you can disapply this change. But I highly recommend you to adding it in both cases as it will increase the logic of your website.
> zeplin: 
![image](https://github.com/touredri/bookstore/assets/91486791/e76c186f-6401-40e8-b6c1-790734946a98)

Reply:
@tugsem Hi,  It is not part of this, they said to replace with author

Reply:
To add a new book, you should enter the title and author, Only. Thanks!